source/hardware/controllers/numark_ns6ii.rst

```diff
@@ -0,0 +1,543 @@
+one. The Tempo Fader uses soft-takeover, meaning if the physical location does
+not match the software one, you will need to synchronize it manually again (this
+avoids sudden jumps). The closer the slider is to its neutral position, the
+brighter the middle LED is. The closer the physical slider position is to the
+software position, the dimmer they get. This allows you to judge how far you
+have to move the slider until you have synchronized the fader and how far you
+have to move it until it is at neutral speed. These leds (with the exceptions of
+the middle LED are also repeated on the Jogwheel LCD Display).
+
+
+Mixer Section
+^^^^^^^^^^^^^
+
+The NS6II features 4 identical (with respect to symmetry) Mixer Channels. See :ref:`interface-eq-gain` for a complete explanation of the individual concepts
+
+.. image:: ../../_static/controllers/numark_ns6ii_schematics/mixer.svg
+   :align: left
+   :width: 20%
+
+
+========  ==============================================================  ==========================================
+No.       Control                                                         Function
+========  ==============================================================  ==========================================
+38        :hwlabel:`LOAD`                                                 Load the currently selected track from the library into the deck corresponding to the channel
+46        :hwlabel:`A` / :hwlabel:`B`                                     Route Channel to left/right :ref:`Effect Unit <interface-effects>`.
+7         :hwlabel:`GAIN`                                                 Channel Gain (see :ref:`interface-eq-gain`)
+9         :hwlabel:`HIGH`                                                 Channel High/Treble EQ
+8         Channel VuMeter                                                 loudness of signal of that specific channel (after EQ, before Filter and Faders). A white LED indicates a clipping signal (see :term:`level meter`).
+10        :hwlabel:`MID`                                                  Channel Mid EQ
+11        :hwlabel:`LOW`                                                  Channel Low/Bass EQ
+47        :hwlabel:`FILTER`                                               This knob lets you control the Meta knob of the deck’s Quick Effect.
+14        :hwlabel:`A OFF B` switch                                       The :term:`crossfader` orientation switches determine to which side of the crossfader the deck is assigned. It can be set to the left or right side, or to the center where it is unaffected by the crossfader knob.
+13        :hwlabel:`🎧` (pfl/cue) button                                  Toggle :term:`PFL`. Notice: due to hardware constraints, this always works in numark mode: clicking a single button solos the corresponding channel, to listen to multiple channels, press and then release multiple buttons at once.
```

Comment:
Umm, nice character but hard to see with the dark button bg.
![image](https://github.com/user-attachments/assets/55794e87-9135-45de-aebf-dea63013e209)

Can't propose an alternative so it's okay I guess

Reply:
yeah, I agree its suboptimal, but the exact icon will depend on the user anyways.